### ISSUE TITLE
"Unable to get profile information from Google  Sign in with Google"

### ISSUE THREAD

**Issue by dalbinaco** [302bis/figma-app-ubuntu#10]
![image](https://user-images.githubusercontent.com/16964673/40616256-bf49dd44-6260-11e8-8dcf-98f9477a6858.png)

**Comment by zamhaq** [comment]
@dalbinaco were you able to fix this?

**Comment by dalbinaco** [comment]
@zamhaq No. So I just went to "Forgot password" on Figma web and now I have an email and password like a regular account. Then I was able to login using email/password.

**Comment by matheszabi** [comment]
https://forum.figma.com/t/unable-to-get-profile-from-google/14575
it doesn't help the logout from desktop app for me
I try to import in Vectornator the Figma plan, but can't get from browser cookies the logged in account.

<img width="565" alt="Screenshot 2022-04-27 at 16 12 38" src="https://user-images.githubusercontent.com/506560/165525946-1b692385-95cc-4a8f-b4a3-94baa9f4c197.png">

**Comment by matheszabi** [comment]
3 year old bug, congrats!

**Comment by mistadikay** [comment]
8 year old!
